package.json

```diff
@@ -79,6 +79,15 @@
       "items": {
         "type": "string"
       }
+    },
+    "silenceWhileTypingRules": {
+      "title": "Silence rules while typing",
```

Comment:
Maybe "Rules to silence while typing"? The current wording implies _all_ rules are silenced while typing, and that this should be a flag.